### ISSUE TITLE
SEVER bug on new 1.3.5 Unexpected token (46:67) You may need an appropriate loader to handle this file type. | import * as import35 

### ISSUE THREAD

**Issue by mburger81** [driftyco/ionic-app-scripts#930]
#### Short description of the problem:
Upgrade to `1.3.5` broke the build script `npm run build --prod`
I'm using `ionic-cli@beta9` and `ionic-angular@3.1.0` but also with other versions I have this problems!

This is the repeating error:
```
[13:47:06]  ionic-app-script task: "build" 
[13:47:06]  Error: ./src/pages/dashboard/dashboard.module.ngfactory.ts Module parse failed: 
            /web-client-ionic/node_modules/@ionic/app-scripts/dist/webpack/optimization-loader.js!/web-client-ionic/src/pages/dashboard/dashboard.module.ngfactory.ts
            Unexpected token (46:67) You may need an appropriate loader to handle this file type. | import * as import35 
            from './dashboard'; | import * as import36 from 'ionic-angular/util/module-loader'; | class 
            DashboardPageModuleInjector extends import0.ɵNgModuleInjector<import1.DashboardPageModule> { | 
            _CommonModule_0:import2.CommonModule; | _ɵba_1:import3.ɵba; @ ./src async @ 
            ./~/ionic-angular/util/ng-module-loader.js @ ./src/app/app.module.ngfactory.js @ 
            ./src/app/main.ts,./src/pages/data-recording/data-recording.module.ngfactory.ts Module parse failed: 
            /web-client-ionic/node_modules/@ionic/app-scripts/dist/webpack/optimization-loader.js!/web-client-ionic/src/pages/data-recording/data-recording.module.ngfactory.ts
            Unexpected token (38:71) You may need an appropriate loader to handle this file type. | import * as import27 
            from './data-recording'; | import * as import28 from 'ionic-angular/util/module-loader'; | class 
            DataRecordingPageModuleInjector extends import0.ɵNgModuleInjector<import1.DataRecordingPageModule> { | 
            _CommonModule_0:import2.CommonModule; | _ɵba_1:import3.ɵba; @ ./src async @ 
            ./~/ionic-angular/util/ng-module-loader.js @ ./src/app/app.module.ngfactory.js @ 
            ./src/app/main.ts,./src/pages/export/export.module.ngfactory.ts Module parse failed: 
            /web-client-ionic/node_modules/@ionic/app-scripts/dist/webpack/optimization-loader.js!/web-client-ionic/src/pages/export/export.module.ngfactory.ts
            Unexpected token (33:64) You may need an appropriate loader to handle this file type. | import * as import22 
            from './export'; | import * as import23 from 'ionic-angular/util/module-loader'; | class 
            ExportPageModuleInjector extends import0.ɵNgModuleInjector<import1.ExportPageModule> { | 
            _CommonModule_0:import2.CommonModule; | _ɵba_1:import3.ɵba; @ ./src async @ 
            ./~/ionic-angular/util/ng-module-loader.js @ ./src/app/app.module.ngfactory.js @ 
            ./src/app/main.ts,./src/pages/heating-control/heating-control.module.ngfactory.ts Module parse failed: 
            /web-client-ionic/node_modules/@ionic/app-scripts/dist/webpack/optimization-loader.js!/web-client-ionic/src/pages/heating-control/heating-control.module.ngfactory.ts
            Unexpected token (38:72) You may need an appropriate loader to handle this file type. | import * as import27 
            from './heating-control'; | import * as import28 from 'ionic-angular/util/module-loader'; | class 
            HeatingControlPageModuleInjector extends import0.ɵNgModuleInjector<import1.HeatingControlPageModule> { | 
            _CommonModule_0:import2.CommonModule; | _ɵba_1:import3.ɵba; @ ./src async @ 
            ./~/ionic-angular/util/ng-module-loader.js @ ./src/app/app.module.ngfactory.js @ 
            ./src/app/main.ts,./src/pages/login/login.module.ngfactory.ts Module parse failed: 
            /web-client-ionic/node_modules/@ionic/app-scripts/dist/webpack/optimization-loader.js!/web-client-ionic/src/pages/login/login.module.ngfactory.ts
            Unexpected token (30:63) You may need an appropriate loader to handle this file type. | import * as import19 
            from './login'; | import * as import20 from 'ionic-angular/util/module-loader'; | class 
            LoginPageModuleInjector extends import0.ɵNgModuleInjector<import1.LoginPageModule> { | 
            _CommonModule_0:import2.CommonModule; | _ɵba_1:import3.ɵba; @ ./src async @ 
            ./~/ionic-angular/util/ng-module-loader.js @ ./src/app/app.module.ngfactory.js @ 
            ./src/app/main.ts,./src/pages/main/main.module.ngfactory.ts Module parse failed: 
            /web-client-ionic/node_modules/@ionic/app-scripts/dist/webpack/optimization-loader.js!/web-client-ionic/src/pages/main/main.module.ngfactory.ts
            Unexpected token (33:62) You may need an appropriate loader to handle this file type. | import * as import22 
            from './main'; | import * as import23 from 'ionic-angular/util/module-loader'; | class 
            MainPageModuleInjector extends import0.ɵNgModuleInjector<import1.MainPageModule> { | 
            _CommonModule_0:import2.CommonModule; | _ɵba_1:import3.ɵba; @ ./src async @ 
            ./~/ionic-angular/util/ng-module-loader.js @ ./src/app/app.module.ngfactory.js @ 
            ./src/app/main.ts,./src/pages/marketplace/marketplace.module.ngfactory.ts Module parse failed: 
            /web-client-ionic/node_modules/@ionic/app-scripts/dist/webpack/optimization-loader.js!/web-client-ionic/src/pages/marketplace/marketplace.module.ngfactory.ts
            Unexpected token (32:69) You may need an appropriate loader to handle this file type. | import * as import21 
            from './marketplace'; | import * as import22 from 'ionic-angular/util/module-loader'; | class 
            MarketplacePageModuleInjector extends import0.ɵNgModuleInjector<import1.MarketplacePageModule> { | 
            _CommonModule_0:import2.CommonModule; | _ɵba_1:import3.ɵba; @ ./src async @ 
            ./~/ionic-angular/util/ng-module-loader.js @ ./src/app/app.module.ngfactory.js @ 
            ./src/app/main.ts,./src/pages/sensor-config/sensor-config.module.ngfactory.ts Module parse failed: 
            /web-client-ionic/node_modules/@ionic/app-scripts/dist/webpack/optimization-loader.js!/web-client-ionic/src/pages/sensor-config/sensor-config.module.ngfactory.ts
            Unexpected token (49:70) You may need an appropriate loader to handle this file type. | import * as import38 
            from './sensor-config'; | import * as import39 from 'ionic-angular/util/module-loader'; | class 
            SensorConfigPageModuleInjector extends import0.ɵNgModuleInjector<import1.SensorConfigPageModule> { | 
            _CommonModule_0:import2.CommonModule; | _ɵba_1:import3.ɵba; @ ./src async @ 
            ./~/ionic-angular/util/ng-module-loader.js @ ./src/app/app.module.ngfactory.js @ 
            ./src/app/main.ts,./src/pages/user-management/user-management.module.ngfactory.ts Module parse failed: 
            /web-client-ionic/node_modules/@ionic/app-scripts/dist/webpack/optimization-loader.js!/web-client-ionic/src/pages/user-management/user-management.module.ngfactory.ts
            Unexpected token (35:72) You may need an appropriate loader to handle this file type. | import * as import24 
            from './user-management'; | import * as import25 from 'ionic-angular/util/module-loader'; | class 
            UserManagementPageModuleInjector extends import0.ɵNgModuleInjector<import1.UserManagementPageModule> { | 
            _CommonModule_0:import2.CommonModule; | _ɵba_1:import3.ɵba; @ ./src async @ 
            ./~/ionic-angular/util/ng-module-loader.js @ ./src/app/app.module.ngfactory.js @ 
            ./src/app/main.ts,./src/pages/videocams-configure/videocams-configure.module.ngfactory.ts Module parse 
            failed: 
            /web-client-ionic/node_modules/@ionic/app-scripts/dist/webpack/optimization-loader.js!/web-client-ionic/src/pages/videocams-configure/videocams-configure.module.ngfactory.ts                                                                                                                                                                                                                                                 
            Unexpected token (37:76) You may need an appropriate loader to handle this file type. | import * as import26 
            from './videocams-configure'; | import * as import27 from 'ionic-angular/util/module-loader'; | class 
            VideoCamsConfigurePageModuleInjector extends import0.ɵNgModuleInjector<import1.VideoCamsConfigurePageModule> 
            { | _CommonModule_0:import2.CommonModule; | _ɵba_1:import3.ɵba; @ ./src async @ 
            ./~/ionic-angular/util/ng-module-loader.js @ ./src/app/app.module.ngfactory.js @ 
            ./src/app/main.ts,./src/components/sensor/sensor-settings/sensor-settings-modal.module.ngfactory.ts Module 
            parse failed: 
            /web-client-ionic/node_modules/@ionic/app-scripts/dist/webpack/optimization-loader.js!/web-client-ionic/src/components/sensor/sensor-settings/sensor-settings-modal.module.ngfactory.ts                                                                                                                                                                                                                                       
            Unexpected token (26:73) You may need an appropriate loader to handle this file type. | import * as import15 
            from './sensor-settings-modal'; | import * as import16 from 'ionic-angular/util/module-loader'; | class 
            SensorSettingsModalModuleInjector extends import0.ɵNgModuleInjector<import1.SensorSettingsModalModule> { | 
            _CommonModule_0:import2.CommonModule; | _ɵba_1:import3.ɵba; @ ./src async @ 
            ./~/ionic-angular/util/ng-module-loader.js @ ./src/app/app.module.ngfactory.js @ 
            ./src/app/main.ts,./src/components/sensor/sensor-settings/sensor-settings-popover.module.ngfactory.ts Module 
            parse failed: 
            /web-client-ionic/node_modules/@ionic/app-scripts/dist/webpack/optimization-loader.js!/web-client-ionic/src/components/sensor/sensor-settings/sensor-settings-popover.module.ngfactory.ts                                                                                                                                                                                                                                     
            Unexpected token (28:75) You may need an appropriate loader to handle this file type. | import * as import17 
            from './sensor-settings-popover'; | import * as import18 from 'ionic-angular/util/module-loader'; | class 
            SensorSettingsPopoverModuleInjector extends import0.ɵNgModuleInjector<import1.SensorSettingsPopoverModule> { 
            | _CommonModule_0:import2.CommonModule; | _ɵba_1:import3.ɵba; @ ./src async @ 
            ./~/ionic-angular/util/ng-module-loader.js @ ./src/app/app.module.ngfactory.js @ ./src/app/main.ts 
Error: ./src/pages/dashboard/dashboard.module.ngfactory.ts
Module parse failed: /web-client-ionic/node_modules/@ionic/app-scripts/dist/webpack/optimization-loader.js!/web-client-ionic/src/pages/dashboard/dashboard.module.ngfactory.ts Unexpected token (46:67)                                                                                                                                                                                                                                   
You may need an appropriate loader to handle this file type.                                                                                                                                                                                       
| import * as import35 from './dashboard';                                                                                                                                                                                                         
| import * as import36 from 'ionic-angular/util/module-loader';                                                                                                                                                                                    
| class DashboardPageModuleInjector extends import0.ɵNgModuleInjector<import1.DashboardPageModule> {                                                                                                                                               
|   _CommonModule_0:import2.CommonModule;                                                                                                                                                                                                          
|   _ɵba_1:import3.ɵba;                                                                                                                                                                                                                            
 @ ./src async                                                                                                                                                                                                                                     
 @ ./~/ionic-angular/util/ng-module-loader.js                                                                                                                                                                                                      
 @ ./src/app/app.module.ngfactory.js                                                                                                                                                                                                               
 @ ./src/app/main.ts,./src/pages/data-recording/data-recording.module.ngfactory.ts                                                                                                                                                                 
Module parse failed: /web-client-ionic/node_modules/@ionic/app-scripts/dist/webpack/optimization-loader.js!/web-client-ionic/src/pages/data-recording/data-recording.module.ngfactory.ts Unexpected token (38:71)                                                                                                                                                                                                                         
You may need an appropriate loader to handle this file type.                                                                                                                                                                                       
| import * as import27 from './data-recording';                                                                                                                                                                                                    
| import * as import28 from 'ionic-angular/util/module-loader';                                                                                                                                                                                    
| class DataRecordingPageModuleInjector extends import0.ɵNgModuleInjector<import1.DataRecordingPageModule> {                                                                                                                                       
|   _CommonModule_0:import2.CommonModule;                                                                                                                                                                                                          
|   _ɵba_1:import3.ɵba;                                                                                                                                                                                                                            
 @ ./src async                                                                                                                                                                                                                                     
 @ ./~/ionic-angular/util/ng-module-loader.js                                                                                                                                                                                                      
 @ ./src/app/app.module.ngfactory.js                                                                                                                                                                                                               
 @ ./src/app/main.ts,./src/pages/export/export.module.ngfactory.ts                                                                                                                                                                                 
Module parse failed: /web-client-ionic/node_modules/@ionic/app-scripts/dist/webpack/optimization-loader.js!/web-client-ionic/src/pages/export/export.module.ngfactory.ts Unexpected token (33:64)                                                                                                                                                                                                                                         
You may need an appropriate loader to handle this file type.                                                                                                                                                                                       
| import * as import22 from './export';                                                                                                                                                                                                            
| import * as import23 from 'ionic-angular/util/module-loader';                                                                                                                                                                                    
| class ExportPageModuleInjector extends import0.ɵNgModuleInjector<import1.ExportPageModule> {                                                                                                                                                     
|   _CommonModule_0:import2.CommonModule;                                                                                                                                                                                                          
|   _ɵba_1:import3.ɵba;                                                                                                                                                                                                                            
 @ ./src async                                                                                                                                                                                                                                     
 @ ./~/ionic-angular/util/ng-module-loader.js                                                                                                                                                                                                      
 @ ./src/app/app.module.ngfactory.js                                                                                                                                                                                                               
 @ ./src/app/main.ts,./src/pages/heating-control/heating-control.module.ngfactory.ts                                                                                                                                                               
Module parse failed: /web-client-ionic/node_modules/@ionic/app-scripts/dist/webpack/optimization-loader.js!/web-client-ionic/src/pages/heating-control/heating-control.module.ngfactory.ts Unexpected token (38:72)                                                                                                                                                                                                                       
You may need an appropriate loader to handle this file type.                                                                                                                                                                                       
| import * as import27 from './heating-control';                                                                                                                                                                                                   
| import * as import28 from 'ionic-angular/util/module-loader';                                                                                                                                                                                    
| class HeatingControlPageModuleInjector extends import0.ɵNgModuleInjector<import1.HeatingControlPageModule> {                                                                                                                                     
|   _CommonModule_0:import2.CommonModule;                                                                                                                                                                                                          
|   _ɵba_1:import3.ɵba;                                                                                                                                                                                                                            
 @ ./src async                                                                                                                                                                                                                                     
 @ ./~/ionic-angular/util/ng-module-loader.js                                                                                                                                                                                                      
 @ ./src/app/app.module.ngfactory.js                                                                                                                                                                                                               
 @ ./src/app/main.ts,./src/pages/login/login.module.ngfactory.ts                                                                                                                                                                                   
Module parse failed: /web-client-ionic/node_modules/@ionic/app-scripts/dist/webpack/optimization-loader.js!/web-client-ionic/src/pages/login/login.module.ngfactory.ts Unexpected token (30:63)                                                                                                                                                                                                                                           
You may need an appropriate loader to handle this file type.                                                                                                                                                                                       
| import * as import19 from './login';                                                                                                                                                                                                             
| import * as import20 from 'ionic-angular/util/module-loader';                                                                                                                                                                                    
| class LoginPageModuleInjector extends import0.ɵNgModuleInjector<import1.LoginPageModule> {                                                                                                                                                       
|   _CommonModule_0:import2.CommonModule;                                                                                                                                                                                                          
|   _ɵba_1:import3.ɵba;                                                                                                                                                                                                                            
 @ ./src async                                                                                                                                                                                                                                     
 @ ./~/ionic-angular/util/ng-module-loader.js                                                                                                                                                                                                      
 @ ./src/app/app.module.ngfactory.js                                                                                                                                                                                                               
 @ ./src/app/main.ts,./src/pages/main/main.module.ngfactory.ts                                                                                                                                                                                     
Module parse failed: /web-client-ionic/node_modules/@ionic/app-scripts/dist/webpack/optimization-loader.js!/web-client-ionic/src/pages/main/main.module.ngfactory.ts Unexpected token (33:62)                                                                                                                                                                                                                                             
You may need an appropriate loader to handle this file type.                                                                                                                                                                                       
| import * as import22 from './main';                                                                                                                                                                                                              
| import * as import23 from 'ionic-angular/util/module-loader';                                                                                                                                                                                    
| class MainPageModuleInjector extends import0.ɵNgModuleInjector<import1.MainPageModule> {                                                                                                                                                         
|   _CommonModule_0:import2.CommonModule;                                                                                                                                                                                                          
|   _ɵba_1:import3.ɵba;                                                                                                                                                                                                                            
 @ ./src async                                                                                                                                                                                                                                     
 @ ./~/ionic-angular/util/ng-module-loader.js                                                                                                                                                                                                      
 @ ./src/app/app.module.ngfactory.js                                                                                                                                                                                                               
 @ ./src/app/main.ts,./src/pages/marketplace/marketplace.module.ngfactory.ts                                                                                                                                                                       
Module parse failed: /web-client-ionic/node_modules/@ionic/app-scripts/dist/webpack/optimization-loader.js!/web-client-ionic/src/pages/marketplace/marketplace.module.ngfactory.ts Unexpected token (32:69)                                                                                                                                                                                                                               
You may need an appropriate loader to handle this file type.                                                                                                                                                                                       
| import * as import21 from './marketplace';                                                                                                                                                                                                       
| import * as import22 from 'ionic-angular/util/module-loader';                                                                                                                                                                                    
| class MarketplacePageModuleInjector extends import0.ɵNgModuleInjector<import1.MarketplacePageModule> {                                                                                                                                           
|   _CommonModule_0:import2.CommonModule;                                                                                                                                                                                                          
|   _ɵba_1:import3.ɵba;                                                                                                                                                                                                                            
 @ ./src async                                                                                                                                                                                                                                     
 @ ./~/ionic-angular/util/ng-module-loader.js                                                                                                                                                                                                      
 @ ./src/app/app.module.ngfactory.js                                                                                                                                                                                                               
 @ ./src/app/main.ts,./src/pages/sensor-config/sensor-config.module.ngfactory.ts                                                                                                                                                                   
Module parse failed: /web-client-ionic/node_modules/@ionic/app-scripts/dist/webpack/optimization-loader.js!/web-client-ionic/src/pages/sensor-config/sensor-config.module.ngfactory.ts Unexpected token (49:70)                                                                                                                                                                                                                           
You may need an appropriate loader to handle this file type.                                                                                                                                                                                       
| import * as import38 from './sensor-config';                                                                                                                                                                                                     
| import * as import39 from 'ionic-angular/util/module-loader';                                                                                                                                                                                    
| class SensorConfigPageModuleInjector extends import0.ɵNgModuleInjector<import1.SensorConfigPageModule> {                                                                                                                                         
|   _CommonModule_0:import2.CommonModule;                                                                                                                                                                                                          
|   _ɵba_1:import3.ɵba;                                                                                                                                                                                                                            
 @ ./src async                                                                                                                                                                                                                                     
 @ ./~/ionic-angular/util/ng-module-loader.js                                                                                                                                                                                                      
 @ ./src/app/app.module.ngfactory.js                                                                                                                                                                                                               
 @ ./src/app/main.ts,./src/pages/user-management/user-management.module.ngfactory.ts                                                                                                                                                               
Module parse failed: /web-client-ionic/node_modules/@ionic/app-scripts/dist/webpack/optimization-loader.js!/web-client-ionic/src/pages/user-management/user-management.module.ngfactory.ts Unexpected token (35:72)                                                                                                                                                                                                                       
You may need an appropriate loader to handle this file type.                                                                                                                                                                                       
| import * as import24 from './user-management';                                                                                                                                                                                                   
| import * as import25 from 'ionic-angular/util/module-loader';                                                                                                                                                                                    
| class UserManagementPageModuleInjector extends import0.ɵNgModuleInjector<import1.UserManagementPageModule> {                                                                                                                                     
|   _CommonModule_0:import2.CommonModule;                                                                                                                                                                                                          
|   _ɵba_1:import3.ɵba;                                                                                                                                                                                                                            
 @ ./src async                                                                                                                                                                                                                                     
 @ ./~/ionic-angular/util/ng-module-loader.js                                                                                                                                                                                                      
 @ ./src/app/app.module.ngfactory.js                                                                                                                                                                                                               
 @ ./src/app/main.ts,./src/pages/videocams-configure/videocams-configure.module.ngfactory.ts                                                                                                                                                       
Module parse failed: /web-client-ionic/node_modules/@ionic/app-scripts/dist/webpack/optimization-loader.js!/web-client-ionic/src/pages/videocams-configure/videocams-configure.module.ngfactory.ts Unexpected token (37:76)                                                                                                                                                                                                               
You may need an appropriate loader to handle this file type.                                                                                                                                                                                       
| import * as import26 from './videocams-configure';                                                                                                                                                                                               
| import * as import27 from 'ionic-angular/util/module-loader';                                                                                                                                                                                    
| class VideoCamsConfigurePageModuleInjector extends import0.ɵNgModuleInjector<import1.VideoCamsConfigurePageModule> {
|   _CommonModule_0:import2.CommonModule;
|   _ɵba_1:import3.ɵba;
 @ ./src async
 @ ./~/ionic-angular/util/ng-module-loader.js
 @ ./src/app/app.module.ngfactory.js
 @ ./src/app/main.ts,./src/components/sensor/sensor-settings/sensor-settings-modal.module.ngfactory.ts
Module parse failed: /web-client-ionic/node_modules/@ionic/app-scripts/dist/webpack/optimization-loader.js!/web-client-ionic/src/components/sensor/sensor-settings/sensor-settings-modal.module.ngfactory.ts Unexpected token (26:73)
You may need an appropriate loader to handle this file type.
| import * as import15 from './sensor-settings-modal';
| import * as import16 from 'ionic-angular/util/module-loader';
| class SensorSettingsModalModuleInjector extends import0.ɵNgModuleInjector<import1.SensorSettingsModalModule> {
|   _CommonModule_0:import2.CommonModule;
|   _ɵba_1:import3.ɵba;
 @ ./src async
 @ ./~/ionic-angular/util/ng-module-loader.js
 @ ./src/app/app.module.ngfactory.js
 @ ./src/app/main.ts,./src/components/sensor/sensor-settings/sensor-settings-popover.module.ngfactory.ts
Module parse failed: /web-client-ionic/node_modules/@ionic/app-scripts/dist/webpack/optimization-loader.js!/web-client-ionic/src/components/sensor/sensor-settings/sensor-settings-popover.module.ngfactory.ts Unexpected token (28:75)
You may need an appropriate loader to handle this file type.
| import * as import17 from './sensor-settings-popover';
| import * as import18 from 'ionic-angular/util/module-loader';
| class SensorSettingsPopoverModuleInjector extends import0.ɵNgModuleInjector<import1.SensorSettingsPopoverModule> {
|   _CommonModule_0:import2.CommonModule;
|   _ɵba_1:import3.ɵba;
 @ ./src async
 @ ./~/ionic-angular/util/ng-module-loader.js
 @ ./src/app/app.module.ngfactory.js
 @ ./src/app/main.ts
    at BuildError.Error (native)
    at new BuildError (/web-client-ionic/node_modules/@ionic/app-scripts/dist/util/errors.js:16:28)
    at /web-client-ionic/node_modules/@ionic/app-scripts/dist/preprocess.js:18:21
    at process._tickCallback (internal/process/next_tick.js:109:7)
```



#### What behavior are you expecting?
It should work


**Which @ionic/app-scripts version are you using?**
1.3.5

**Other information:** (e.g. stacktraces, related issues, suggestions how to fix, stackoverflow links, forum links, etc)

**Comment by danbucholtz** [comment]
Try `nightly`. `npm install @ionic/app-scripts@nightly`. I'll push a fix for this today.

Thanks,
Dan

**Comment by mburger81** [comment]
Using last nightly `@1.3.5-201704271445` seems to resolve this bug. I will wait to close this bug for stable release and check before if anything is okay.

**Comment by danbucholtz** [comment]
Yep, I will publish the fix today. Sorry about that. I merged a ton of PRs and did not personally run into this until after I published.

Thanks,
Dan

**Comment by mburger81** [comment]
I'm happy to help you!!

If it is possibile please have a look at my other bug I opened today #927 this is blocking us from developement! It's a very strange bug.

**Comment by danbucholtz** [comment]
This should be fixed in `1.3.6`. `npm install @ionic/app-scripts@latest`

Thanks,
Dan
